lib/fs_utils/generate.js

```diff
@@ -188,6 +188,11 @@ const optimize = (data, map, path, optimizers, sourceFiles) => {
   return promiseReduce(optimizers, runOptimizer, initial);
 };
 
+const getInlineSourceMapUrl = map => {
+  const base64String = new Buffer(JSON.stringify(map)).toString('base64');
+  return `data:application/json;charset=utf-8;base64,${base64String}`;
```

Comment:
> might specify a charset parameter

Personally, I have always omitted `charset` for HTML/CSS/JS and never encountered problems due to lack of it.